packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.kt

```diff
@@ -56,7 +56,7 @@ import kotlin.math.min
     message = "This class is part of Legacy Architecture and will be removed in a future release",
     level = DeprecationLevel.WARNING,
 )
-public class ReactTextShadowNode
+public open class ReactTextShadowNode
```

Comment:
I know there is feature freeze on old arch code, but would be very appreciated if we can get that one in.

Reply:
This class is going to be deleted in 0.83+ and beyond so I'd suggest we don't make it open.

Instead this shadow node should probably be copied over and maintained by the library